apps/dashboard/src/app/(app)/team/[team_slug]/(team)/~/support/apis/feedback.ts

```diff
@@ -0,0 +1,62 @@
+"use server";
+
+interface FeedbackData {
+  rating: number;
+  feedback: string;
+  ticketId?: string;
+}
+
+export async function submitSupportFeedback(
+  data: FeedbackData,
+): Promise<{ success: true } | { error: string }> {
+  try {
+    // Fail fast on missing configuration
+    const siwaUrl =
+      process.env.SIWA_URL ?? process.env.NEXT_PUBLIC_SIWA_URL ?? "";
+    if (!siwaUrl) {
+      throw new Error("SIWA URL not configured");
+    }
+
+    const apiKey = process.env.SERVICE_AUTH_KEY_SIWA;
+    if (!apiKey) {
+      throw new Error("SERVICE_AUTH_KEY_SIWA not configured");
+    }
+
+    // Basic input validation/normalization
+    if (!Number.isFinite(data.rating) || data.rating < 1 || data.rating > 5) {
+      return { error: "Rating must be an integer between 1 and 5." };
+    }
+
+    const normalizedFeedback = (data.feedback ?? "")
+      .toString()
+      .trim()
+      .slice(0, 1000); // hard cap length
+
+    const payload = {
+      rating: Math.round(data.rating),
```

Comment:
to make all this math stuff easier just send a string instead of a number for the rating "0", "1", "2", etc yu'll have to do way less logic around numbers here